inst/odin/lancelot.R

```diff
@@ -170,8 +173,48 @@ delta_new_conf <-
   sum(n_W_D_unconf_to_conf)
 update(cum_new_conf) <- cum_new_conf + delta_new_conf
 
+
+delta_all_admission_0_9_conf <-
+  sum(delta_diagnoses_admitted[1:2, ])
+
+delta_all_admission_10_19_conf <-
+  sum(delta_diagnoses_admitted[3:4, ])
+
+delta_all_admission_20_29_conf <-
+  sum(delta_diagnoses_admitted[5:6, ]) +
+  sum(delta_diagnoses_admitted[18, ]) * 1 / 8
+
+delta_all_admission_30_39_conf <-
+  sum(delta_diagnoses_admitted[7:8, ]) +
+  sum(delta_diagnoses_admitted[18, ]) * 1 / 8
+
+delta_all_admission_40_49_conf <-
+  sum(delta_diagnoses_admitted[9:10, ]) +
+  sum(delta_diagnoses_admitted[18, ]) * 2 / 8
+
+delta_all_admission_50_59_conf <-
+  sum(delta_diagnoses_admitted[11:12, ]) +
+  sum(delta_diagnoses_admitted[18, ]) * 3 / 8
+
+delta_all_admission_60_69_conf <-
+  sum(delta_diagnoses_admitted[13:14, ]) +
+  sum(delta_diagnoses_admitted[18, ]) * 1 / 8 +
```

Comment:
this probably won't make a huge deal of difference, but we have admissions taken from CHW as follows (compare with proportion of CHW drawn from that group in brackets)
20-29: 1/8 (1/8)
30-39: 1/8 (2/8)
40-49: 2/8 (2/8)
50-59: 3/8 (2/8)
60-69: 1/8 (1/8)
Do we want that mismatch?

Reply:
I've changed it now